tests/kafkatest/tests/streams/streams_broker_down_resilience_test.py

```diff
@@ -112,3 +116,68 @@ def test_streams_resilient_to_broker_down(self):
         self.assert_produce_consume("after_broker_stop")
 
         self.kafka.stop()
+
+    def test_streams_runs_with_broker_down_initially(self):
+        self.kafka.start()
+        node = self.kafka.leader(self.inputTopic)
+        self.kafka.stop_node(node)
+
+        broker_down_initially_in_seconds = 60
+
+        configs = self.get_configs(extra_configs=",application.id=starting_wo_broker_id")
+
+        # start streams with broker down initially
+        processor = StreamsBrokerDownResilienceService(self.test_context, self.kafka, configs)
+        processor.start()
+
+        processor_2 = StreamsBrokerDownResilienceService(self.test_context, self.kafka, configs)
+        processor_2.start()
+
+        processor_3 = StreamsBrokerDownResilienceService(self.test_context, self.kafka, configs)
+        processor_3.start()
+
+        time.sleep(broker_down_initially_in_seconds)
+
+        # now start broker
+        self.kafka.start_node(node)
+        # give broker time to start up
+        time.sleep(20)
+
+        # assert streams can process when starting with broker down
+        self.assert_produce_consume("running_with_broker_down_initially", num_messages=9)
+
+        self.kafka.stop()
+
+    def test_streams_shuts_down_clean_with_broker_down(self):
```

Comment:
I left couple of comments below. The name indicate that we want to test streams shutdown -- thus, I am not sure why we restart the broker at the end of the test? Or is the goal of the test different?

Reply:
The goal of the test is to shut down streams nodes of a multiple node deployment while broker down, then bring broker back up and confirm remaining stream(s) apps can function, getting clarification, however.